### PR TITLE
Fixed proposing TLD

### PR DESCRIPTION
**Pull request by mchf**
bnc#923990

Also a fix for bnc#858908 (use /etc/hostname instead of /etc/HOSTNAME)

### CODE REVIEW

File: package/yast2.changes [hunk 1]
```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Mon Jun 29 07:35:21 UTC 2015 - mfilka@suse.com
+
+- bnc#858908
+  - use /etc/hostname instead of /etc/HOSTNAME
+- bnc#923990
```
Comment (mvidner) [inline package/yast2.changes]: Also https://bugzilla.suse.com/show_bug.cgi?id=922765 please

**Comment by mvidner** [comment]
Just the changelog, otherwise LGTM

**Comment by mchf** [comment]
Changelog updated, going to merge once travis is done.

Thanks for review.

File: library/types/testsuite/tests/Hostname.out [hunk 2]
```diff
@@ -41,4 +41,4 @@ Return	true
 Dump	Hostname::SplitFQ
 Return	["ftp", "suse.cz"]
 Return	["123", "com"]
-Return	["beholder"]
+Return	["beholder", ""]
```
Comment (kobliha) [inline library/types/testsuite/tests/Hostname.out]: Guys, is this actually correct?

**Pull request merged**